modules/reference/pages/properties/broker-properties.adoc

```diff
@@ -203,15 +178,24 @@ A number that uniquely identifies the broker within the cluster. If `null` (the
 
 CAUTION: The `node_id` property must not be changed after a broker joins the cluster.
 
-
 *Accepted values:* [`0`, `4294967295`]
 
 *Type:* integer
 
-*Optional:* No
+*Visibility:* `user`
+
+*Default:* `null`
+
+---
+
+=== node_id_overrides
+
+List of node ID and UUID overrides to be applied at broker startup. Each entry includes the current UUID and desired ID and UUID. Each entry applies to a given node if and only if 'current' matches that node's current UUID.
```

Comment:
Each entry includes the current UUID and desired ID and UUID
^^ this says that each entry includes the UUID...twice.

Shouldn't it be: 
Each entry includes the current UUID, and desired ID and UUID

Reply:
property introduced in https://github.com/redpanda-data/redpanda/pull/22972 per @oleiman .

Reply:
```suggestion
A list of overrides for node IDs and UUIDs applied when the broker starts. Each entry specifies the current UUID and the desired new ID and UUID. The override is applied only if the current UUID matches the node's actual UUID.
```

Reply:
@oleiman WDYT?

Reply:
Resolved not going to include in public docs.